server/services/homepageService.ts

```diff
@@ -40,4 +40,9 @@ export default class HomepageService {
       await this.keyWorkerApiClientBuilder(clientToken).getPrisonMigrationStatus(activeCaseLoadId)
     return prisonMigrationStatus
   }
+
+  public async getStaffRoles(clientToken: string, activeCaseLoadId: string, staffId: number) {
+    const staffRoles = await this.prisonApiClientBuilder(clientToken).getStaffRoles(staffId, activeCaseLoadId)
+    return staffRoles
+  }
```

Comment:
```suggestion
  public getStaffRoles(clientToken: string, activeCaseLoadId: string, staffId: number) {
    return this.prisonApiClientBuilder(clientToken).getStaffRoles(staffId, activeCaseLoadId)
  }
```

Reply:
redundant variable and can just pass the promise straight through from clent rather than wrapping in another async function which also wraps in a promise